app/scripts/modules/kubernetes/src/loadBalancer/configure/wizard/basicSettings.html

```diff
@@ -83,7 +83,7 @@
       </div>
       <div class="form-group">
         <div class="col-md-9 col-md-offset-3" ng-if="form.loadBalancerName.$error.maxlength">
-          <validation-error message="Load Balancer name can only be 24 characters."></validation-error>
+          <validation-error message="Load Balancer name can only be 63 characters."></validation-error>
```

Comment:
Small nit, can you break `63` out into a variable so that we have to change it in fewer places if necessary?

Reply:
@skannengiesser making sure you saw the above. Could you address?

Reply:
@ethanfrogers Yes, I saw your change request. Sorry, haven't had time so far and it might take me some time to figure out how to optimally do this. Just for clarification: You're asking me to have one shared variable for these 2 validation limits, which means across k8s v1 and k8s v2 providers? Maybe you can point me to where I should define it? Thanks a lot.

Reply:
Yes, exactly. Honestly, I think it's fine if you just do it in this file. I just want to be sure that if we change the number to, say, `64` that I don't have to change it in more than one place.

Reply:
@ethanfrogers Sorry, as I'm having difficulties in setting up deck (and other necessary services) to be able to verify any changes, I don't actually dare changing anything like this. Putting a variable up for these two occurences shouldn't be too difficult, but I wouldn't feel good changing this without being able to verify.

Reply:
Completely agree that ideally this constant would be defined once in this file, going to merge this for now since hopefully we can get rid of this angular template altogether in the nearish future, and also since this change doesn't make it worse (the max # was already defined in multiple spots)